data/json/items/comestibles/meat_dishes.json

```diff
@@ -775,6 +775,21 @@
     "fun": 10,
     "vitamins": [ [ "vitA", 3 ], [ "vitC", 10 ], [ "calcium", 28 ], [ "iron", 28 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "pizza_supreme",
+    "name": "supreme pizza",
+    "copy-from": "pizza_meat",
+    "looks-like": "pizza_meat",
+    "color": "light_red",
+    "weight": "300 g",
+    "calories": 641,
+    "description": "A supreme pizza with ALL the toppings.",
+    "price": 1500,
+    "material": [ "wheat", "flesh", "veggy", "milk" ],
+    "fun": 20,
```

Comment:
This fun score is pretty out of line, pizza_meat is already an extremely high fun score. I'd suggest a Supreme pizza might be 11. Adding some peppers and toppings is great, but it doesn't turn an already extremely tasty food into something on a par with a shot of heroin.

Reply:
Fair enough! I'd taken the fun score from other high-effort foods like the Milkshake but was still a little unsure on where exactly to place it.